python/sglang/bench_serving.py

```diff
@@ -627,6 +627,61 @@ def sample_random_requests(
     return input_requests
 
 
+def gen_prompt(tokenizer, token_num):
+    """Generate a random prompt of specified token length using tokenizer vocabulary."""
+    all_available_tokens = list(tokenizer.get_vocab().values())
```

Comment:
Nit: Using tokenizer.get_vocab().values() may include special tokens, which could cause issues with the generated text.